### ISSUE TITLE
Cannot read property 'textContent' of undefined

### ISSUE THREAD

**Issue by sank3** [Dogfalo/materialize#5766]
![image](https://user-images.githubusercontent.com/34004129/37923056-cf52c980-314b-11e8-99ea-6acd44efc504.png)

Getting undefined error when using autocomplete in Android touch device. 

My Code-
```
$('#tdesc').autocomplete({
		data: {
			"#Breakfast":null,
			"#Cab":null,
			"#ElectricityBill":null,
		},
		limit: 10, // The max amount of results that can be shown at once. Default: Infinity.
		minLength: 1, // The minimum length of the input for the autocomplete to start. Default: 1.
	});
```

## Current Behavior
When dropdown suggestions are populated in input autocomplete, choosing any of them is throwing error though it is setting correct value in the text field. Error is not expected.

![image](https://user-images.githubusercontent.com/34004129/37923056-cf52c980-314b-11e8-99ea-6acd44efc504.png)

## Steps to Reproduce (for bugs)

https://codepen.io/sank3/pen/pLdeRX
Check this from touch devices

## Your Environment

* Version used: [Beta 1.0.0](https://github.com/Dogfalo/materialize/tree/1.0.0-beta)
* Browser Name and version: Cordova/WebViewFactory: com.android.chrome version 65.0.3325.109 (code 332510902)
* Operating System and version (desktop or mobile): Android 7.0 build NPNS25.137-92-4

**Comment by acburst** [comment]
can you try creating a codepen that reproduces this issue?

**Comment by sank3** [comment]
@acburst https://codepen.io/sank3/pen/pLdeRX Issue is observed only on touch devices

**Comment by acburst** [comment]
I have updated the handlers for autocomplete for another issue and I think it should fix this.

Fixed in 553fb8f7

**Comment by ChildishGiant** [comment]
[My site](https://childishgiant.github.io/perdola/) has this on autocomplete scrollbars (Even on 1.0.0-rc.2)
